src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
@alexeevdv, no CMake is a build system. For organizing which files to build. If the preprocessor will parse the file just to decide it will not be built because you are not building for the platform in which that file is relevant, you don't need it to even get to the preprocessor, you already know from the outside you don't want to build that file.
The file is clearly names "PlatformUnix", and that's because all of the file is Unix specific. You don't need to get to the preprocessor stage to know you don't want it built on Windows and you want it built on POSIX based OSes. CMake already knows that.